doc/OnlineDocs/contributed_packages/pyros.rst

```diff
@@ -397,22 +397,23 @@ If we choose to designate all variables as either design or state variables, wit
   ...                                  options = {
   ...                                     "objective_focus": pyros.ObjectiveType.worst_case,
   ...                                     "solve_master_globally": True
-  ...                                   })
+  ...                                   }) # doctest: +ELLIPSIS
   ===========================================================================================
   PyROS: Pyomo Robust Optimization Solver ...
   ===========================================================================================
   ...
   INFO: Robust optimal solution identified. Exiting PyROS.
+  INFO: Objective: 4836738...
   ...
 
   >>> # === Print results ===
   >>> single_stage_final_objective = pyo.value(results_1.final_objective_value)
-  >>> print("Final objective value: %s" % single_stage_final_objective) # doctest: +ELLIPSIS
+  >>> print("Final objective value: %s" % single_stage_final_objective)
   Final objective value: ...
-  >>> print("PyROS termination condition: %s" % results_1.grcs_termination_condition) # doctest: +ELLIPSIS
+  >>> print("PyROS termination condition: %s" % results_1.grcs_termination_condition)
   PyROS termination condition: ...
 
-The above code yields: "Final objective value: 48,349,406.72"
+The above code yields: "Final objective value: 48,367,381"
```

Comment:
If you put the value into the doctest above, then you can remove this (and remove the possibility that the value becomes incorrect)

Reply:
I see, I will do that.